README.md

```diff
@@ -842,6 +843,18 @@ directly.
 └────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
 ```
 
+## Reliability Guarantees
+
+### Events are not durable
+
+The event received by Event Gateway is stored only in memory, it's not persisted to disk during processing. This means that in case of hardware failure or software crash the event may not be delivered to the subscriber. For a synchronous subscription (`http` or `invoke` event) it can manifest by error message returned to the requester. If there is multiple subscribes to the same custom event type, in case of failure the event may not be delivered to all of them.
```

Comment:
If there is hardware failure or software crash nothing is returned from EG because of the obvious reason. What I meant here was that client will get some kind of error which depends on the setup e.g. for locally hosted EG it will be simple connection error, for self hosted EG behind AWS ELB there will be error from ELB etc.

Reply:
I understand, but we should have the codes and messages clearly documented - Is that in the API docs?